bundles/persistence/org.openhab.persistence.mysql/java/org/openhab/persistence/mysql/internal/MysqlPersistenceService.java

```diff
@@ -421,15 +421,20 @@ public void store(Item item, String alias) {
         String sqlCmd = null;
         PreparedStatement statement = null;
         try {
+            // Get current timestamp
+            long timeNow = Calendar.getInstance().getTimeInMillis();
+    	    java.sql.Timestamp timestamp = new java.sql.Timestamp(timeNow);
```

Comment:
Indentation (consider using the Eclipse IDE to auto-format code).  Also, please import java.sql.Timestamp and use here without package qualifier.